scripts/package.py

```diff
@@ -46,7 +47,12 @@ def build_pex_pex(output_file: PurePath, verbosity: int = 0) -> None:
     subprocess.run(args, check=True)
 
 
-def describe_git_rev() -> str:
+def describe_rev() -> str:
+    if not os.path.isdir(".git") and os.path.isfile("PKG-INFO"):
```

Comment:
This was always a bug, I just never tried unpacking our sdist and doing a build from there before. That now works.